### PR TITLE
Updated readme to include dependencies hidden with build tags

### PR DESCRIPTION
**Pull request by bcokert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ The `common` package contains our ECS implementations of common game development
   3. If you're on OSX, you will also need Git. You can find instructions [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git#Installing-on-Mac). You can also use homebrew to install git as well. [Open an issue if you have any issues](https://github.com/EngoEngine/engo/issues/new)
 2. Then, you can go get it:
 `go get -u engo.io/engo`
+  1. You may also want to get the dependencies of specific builds, so that build tools like godef can use them:
```
Comment (otraore) [inline README.md]: Can you change "specific builds" to "platform specific builds"

**Comment by bcokert** [comment]
@otraore Updated

**Comment by otraore** [comment]
Thanks for your contribution!

**Comment by bcokert** [comment]
No problem :) there will probably be more coming

**Comment by otraore** [comment]
Looking forward to it 😄

**Pull request merged**